src/pages/developing/web-components-tutorial/step-2.mdx

```diff
@@ -55,13 +60,205 @@ git checkout -b web-components-step-2 upstream/web-components-step-2
+            >Make fast and accurate data-based
+            decisions
+          </dds-leadspace-search-block-heading
+            >
+        </dds-leadspace-block-content>
+        <dds-search-with-typeahead
+          slot="search"
+          leadspace-search></dds-search-with-typeahead>
+      </dds-leadspace-with-search>
+    </div>
+  </div>
+</div>
+```
+<!-- prettier-ignore-end -->
+
+Nice! You have just added the `Leadspace with search` component to the page! But wait...the color of the component doesn't match the design.
+Looking at the props table in the documentation, there is a `adjacent-theme` property we can pass to the component to adjust the theme.
+
+Let's do that by setting `adjacent-theme="g90-and-g100"` in `dds-leadspace-with-search`:
+
+<!-- prettier-ignore-start -->
+```html path=src/index.html
+  <dds-leadspace-with-search adjacent-theme="g90-and-g100">
+```
+<!-- prettier-ignore-end -->
+
+Now `Leadspace with search` should display the proper theme. Great job! Let's move on to the next component.
+
+## Add Table of contents - horizontal
+
+`Table of contents` is a great component to structure the page. We'll be using the horizontal variation in this step.
+
+Let's added the import to the `app.js` file:
+
+```javascript path=src/assets/js/app.js
+import '@carbon/ibmdotcom-web-components/es/components/table-of-contents/index.js';
+```
+
+Next we'll lay the foundation for the next two components that will be incorporated within the `Table of contents`. Add the following
+below the `Leadspace with search` component.
+
+<!-- prettier-ignore-start -->
+```html path=src/index.html
+  <dds-table-of-contents toc-layout="horizontal" stickyOffset="48">
+    <a name="1" data-title="First Item"></a>
+    <h1>I am the first item in the table of contents</h1>
+  </dds-table-of-contents>
+```
+<!-- prettier-ignore-end -->
+
+You should see the `Table of contents` with one item listed. Each item in the `Table of contents` will need a preceding anchor element
+with a unique `name` and title of the item as`data-title.` That's it!
+
+## Add Card section offset
+
+Now within the `Table of contents`, we'll set up the `Card section offset` component. This component requires image assets that are provided
+in the `src/assets/images` folder.
+
+Import the following in `app.js`:
+
+```javascript path=src/assets/js/app.js
+import '@carbon/ibmdotcom-web-components/es/components/card-section-offset/index.js';
+```
+
+Then replace the following within the `Table of contents`:
+
+<!-- prettier-ignore-start -->
+```html path=src/index.html
+    <a name="1" data-title="First Item"></a>
+    <h1>I am the first item in the table of contents</h1>
+```
+<!-- prettier-ignore-end -->
+
+With the `Card section offset`, noting the anchor element precedding that is included for the `Table of contents`:
+
+<!-- prettier-ignore-start -->
+```html path=src/index.html
+<a name="1" data-title="Main benefits of analytics"></a>
```

Comment:
Maybe we can add something like. Since this is the 3rd anchor in the design we can assign name="3"
```suggestion
<a name="3" data-title="Main benefits of analytics"></a>
```